scripts/azure_set_gate_variable_cloud_account_id.ps1

```diff
@@ -45,6 +45,5 @@ if (Test-Path $path) {
     Write-Host "##vso[task.setvariable variable=Sku;isOutput=true]$sku"
     Write-host "The value of Variable IsEnabled is updated to True and output variable ImageUrl to $uri"
 } else {
-    #Write-Host "Artifact path $path does NOT exist for $Version"
-    throw "Artifact path does NOT exist for $Version" # Throwing error if there's no baseimage, instead of just writing it to the host.
+    Write-Host "Artifact path $path does NOT exist for $Version"
```

Comment:
Why have you removed the throw?

Reply:
Hi @robe070 where do you see this code? I added this but removed it very next day, I verified twice and couldn't see on both feature branch. this is must be old.